docs/src/index.md

```diff
@@ -194,7 +194,7 @@ This component follows [JSON Schema](http://json-schema.org/documentation.html)
 
     You can also use `oneOf` with [schema dependencies](dependencies.md#schema-dependencies) to dynamically add schema properties based on input data.
 
-    The `allOf` keyword is supported; it uses [json-schema-merge-allof](https://github.com/mokkabonna/json-schema-merge-allof) to merge subschemas to render the final combined schema in the form. When these subschemas are incompatible, though (or if the library has an error merging it), the `allOf` keyword is dropped from the schema. 
+    The `allOf` keyword is partually supported. 
```

Comment:
partially

Reply:
thanks ,

typo will be fixed in next commit, looking into this not working with a much more complicated schema, I aim to fix this today.

@epicfaace please see my comment on #1623, and take a look at the comment with regards to how schemas are now merged when rendering, while this is still not technically correct, it is much closer in my mind to the json schema specification.